quickstarts/golang/httprouter/config.yml

```
@@ -1,21 +1,39 @@
+  
+  HTTPRouter Golang monitoring provides a new level of application visibility. The Golong-based HTTPRouter also focuses on performance and light resource usage.
 
-  Go-based HTTP request router built with a focus on performance and light
-  resource usage.
 
+  ### Monitoring HTTPRouter
+  
+  New Relic’s HTTPRouter Golang monitoring quickstart helps development teams set up monitoring tools and dashboards to quickly identify and fix errors in their Golang applications. As a Golang developer, you can create custom queries, leverage user-related data to optimize processes, and share highly visual interactive displays of data.
 
-  ## Get started!
 
-  Leverage community expertise and instantly get value out of your telemetry data. This quickstart automatically instruments HTTPRouter with the New Relic Go agent, and allows you to further leverage New Relic's APM capabilities by setting up [custom dashboards](https://docs.newrelic.com/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/), [errors inbox](https://docs.newrelic.com/docs/apm/apm-ui-pages/errors-inbox/errors-inbox/), [transaction tracing](https://docs.newrelic.com/docs/apm/transactions/transaction-traces/introduction-transaction-traces/), and [service maps](https://docs.newrelic.com/docs/understand-dependencies/understand-system-dependencies/service-maps/introduction-service-maps/).
+  ### New Relic and HTTPRouter features
+  
+  Track Golang language apps and microservices with [New Relic’s HTTPRouter and Golang](https://docs.newrelic.com/docs/apm/agents/go-agent/get-started/introduction-new-relic-go/) monitoring quickstart, including throughput, transaction errors, and response times. 
 
-  ## More info
+  With a clear view into garbage collection behavior, memory usage, and CPU usage over time, your Golang development team can better understand your application’s runtime health. HTTPRouter infrastructure monitoring also provides a comprehensive view of host and server data.
+
+  **Key features include:**
+  
+  - Browser monitoring to track browser performance and usage
+  - Cross-application tracing to monitor transactions between APM-monitored apps
+  - Deployment markers for tracking code changes and overall impact on application health and performance
+  - Distributed tracing to better understand how your services and microservices interact
+  - Monitoring Goroutine counts for possible Goroutine leaks
+  - Providing real-time alerting notifications for errors or problems before they impact users
+  - Tracing asynchronous applications and creating segments in multiple Goroutines
+  - Synthetic transaction tracing to connect requests from synthetic monitors to underlying APM transactions
+  - Tracking important metrics by building custom dashboards
+
+New Relic's complete monitoring quickstart helps Golang development teams set up monitoring tools and dashboards to minimize complexity and improve efficiency, and ensure uptime. 
```

This line should be indented to align with the rest of the `description` key:value, Yaml is treating this as a separate key:value because of this.

```suggestion
  New Relic's complete monitoring quickstart helps Golang development teams set up monitoring tools and dashboards to minimize complexity and improve efficiency, and ensure uptime. 
```